data/tlg0014/tlg009/tlg0014.tlg009.perseus-eng2.xml

```diff
@@ -58,7 +63,6 @@
         <profileDesc>
             <langUsage>
                 <language ident="eng">English</language>
```

Comment:
Why is Greek being removed here? I think it is in the notes.

Reply:
hi @lcerrato I thought I had looked through the whole file and not found any Greek, but it can go right back in.  At least this time I managed to open a pull request!

Reply:
Nearly all of the Demosthenes translations have Greek in the notes (there are a few exceptions here and there) so I would not bother removing this from the batch. It's not worthwhile to go through and remove it from the exceptions. There is also Latin in at least one file.

Reply:
@AlisonBabeu  I think the Greek tag was removed from all of these, yes? I would add them all back in, right?

Reply:
I didn't notice it until file 009 and then I thought to check it.